### ISSUE TITLE
ERROR: could not find pathkey item to sort

### ISSUE THREAD

**Issue by hlinnaka** [greenplum-db/gpdb#10729]
### Greenplum version or build
master. Works on 6X_STABLE.

### Step to reproduce the behavior

```
postgres=# create table foo (t text);
NOTICE:  Table doesn't have 'DISTRIBUTED BY' clause -- Using column named 't' as the Greenplum Database data distribution key for this table.
HINT:  The 'DISTRIBUTED BY' clause determines the distribution of data. Make sure column(s) chosen are the optimal data distribution key to minimize skew.
CREATE TABLE
postgres=# set enable_hashagg=off;
SET
postgres=# with cte as 
( select t
  from foo
  group by t
)
select 'xx'::text
from cte
group by 'xx'::text;
ERROR:  could not find pathkey item to sort (createplan.c:6326)
```

**Comment by guofengrichard** [comment]
This issue has been fixed by PR #10759 . We can close it now.

**Comment by hlinnaka** [comment]
Thanks for checking. Closing
